callautomation-Simple-IVR/program.py

```diff
@@ -0,0 +1,175 @@
+                                 + event.kind)                         
+             call_Connection = calling_Automation_client.get_call_connection(event.call_connection_id)            
+             call_Connection_Media =call_Connection.get_call_media()            
+             if event.__class__ == CallConnected:
+                 Logger.log_message(Logger.INFORMATION,'CallConnected event received for call connection id --> ' 
+                                 + event.call_connection_id)                
+                   
+                 recognize_Options = CallMediaRecognizeDtmfOptions(identifier_from_raw_id(request.query_string),max_tones_to_collect=1)
+                 recognize_Options.interrupt_prompt = True
+                 recognize_Options.inter_tone_timeout = 30                 
+                 recognize_Options.initial_silence_timeout=5 
+                 File_source=FileSource(uri=(BaseUri + configuration_manager.get_app_settings('MainMenuAudio')))                 
+                 File_source.play_source_id= 'MainMenu'                 
+                 recognize_Options.play_prompt = File_source                
+                 recognize_Options.operation_context= 'MainMenu'             
+                 call_Connection_Media.start_recognizing(recognize_Options)
+                 
+             if event.__class__ == RecognizeCompleted and event.operation_context == 'MainMenu' :
+                 Logger.log_message(Logger.INFORMATION,'RecognizeCompleted event received for call connection id --> '+ event.call_connection_id
+                                    +'Correlation id:'+event.correlation_id)
+                 toneDetected=event.collect_tones_result.tones[0]
+                 if toneDetected == DtmfTone.ONE:
+                     playSource = FileSource(uri=(BaseUri + configuration_manager.get_app_settings('SalesAudio')))
+                     playSource.play_source_id='SimpleIVR'
+                     PlayOption = call_Connection_Media.play_to_all(playSource,operation_context='SimpleIVR')
+                    
+                 elif toneDetected == DtmfTone.TWO :
+                       playSource = FileSource(uri=(BaseUri + configuration_manager.get_app_settings('MarketingAudio')))
+                       PlayOption = call_Connection_Media.play_to_all(playSource,operation_context='SimpleIVR')
+                 elif toneDetected == DtmfTone.THREE :
+                       playSource = FileSource(uri=(BaseUri + configuration_manager.get_app_settings('CustomerCareAudio')))
+                       PlayOption = call_Connection_Media.play_to_all(playSource, operation_context='SimpleIVR')
+                 elif toneDetected == DtmfTone.FOUR :
+                       playSource = FileSource(uri=(BaseUri + configuration_manager.get_app_settings('AgentAudio')))
+                       PlayOption = call_Connection_Media.play_to_all(playSource,  operation_context='AgentConnect')
```

Comment:
playOption is not required,
also the PlaySource should be in the right format play_source